### ISSUE TITLE
side-nav-item component inside ui-shell is not accessibility compliance.

### ISSUE THREAD

**Issue by shroffsmiti** [IBM/carbon-components-angular#1162]
ibm-sidenav-item does not set the roles correctly for menu-item. 
Here https://github.com/IBM/carbon-components-angular/blob/7f9ebb78a41ea29344b80e0343e3d3bfab89f36d/src/ui-shell/sidenav/sidenav-item.component.ts#L22

It violates accessibility rule:
>  The WAI-ARIA role(s) and/or attribute(s) must be valid for the relevant element.

**Comment by Donisius** [comment]
Hello @shroffsmiti!, I am using all the rulesets available in the latest DAP rule archive but I cant seem to come across this violation, which rule sets are you using?

**Comment by shroffsmiti** [comment]
this is the rule
> HAAC_Aria_Native_Host_Sematics
which u can find here https://aat.w3ibm.mybluemix.net/tools/auth/rules.html?accountId=f154a93a-a580-4a2d-ac62-62262fe6d9f5
@Donisius

**Comment by ninja511** [comment]
closing pending additional information: can't reproduce
